### PR TITLE
[M1P-13] Finish helper testing

### PR DESCRIPTION
**Pull request by adentong**
Finish helper testing and some formatting

#changelog Finish helper testing and some formatting

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Test

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

**Comment by adentong** [comment]
@mccottry Could you take a look at this? Trying to merge this in before Wednesday.

File: app/code/community/Bolt/Boltpay/Helper/Data.php [hunk 1]
```diff
@@ -147,8 +147,6 @@ public function buildOnCheckCallback($checkoutType, $isVirtualQuote = false)
      */
     public function buildOnSuccessCallback($successCustom, $checkoutType)
     {
-        $saveOrderUrl = $this->getMagentoUrl("boltpay/order/save/checkoutType/$checkoutType");
```
Comment (matt-thomason) [inline app/code/community/Bolt/Boltpay/Helper/Data.php]: huh what was this used for, weird

File: app/code/community/Bolt/Boltpay/Helper/Data.php [hunk 2]
```diff
@@ -198,8 +197,6 @@ public function buildOnCloseCallback($closeCustom, $checkoutType)
                     ";
                 break;
             case Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_PRODUCT_PAGE:
-                $quoteId = Mage::getSingleton('checkout/session')->getQuoteId();
```
Comment (matt-thomason) [inline app/code/community/Bolt/Boltpay/Helper/Data.php]: This is not needed anymore? @adentong @mccottry

Comment (adentong) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: These are literally declared and then we just return. So definitely not used.

Comment (matt-thomason) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: sg then, but I'd like @mccottry to weigh in as well 👍

File: app/code/community/Bolt/Boltpay/Helper/Data.php (hunk 1 above)
Comment (mccottry) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: @matt-thomason 1.x version of the plugin used that

File: app/code/community/Bolt/Boltpay/Helper/Data.php (hunk 2 above)
Comment (mccottry) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: @matt-thomason and @adentong 
The code above is unused and is wrong.  There is no meaningful session quote id in the context of product page checkout. What this code represents, (i.e. the close callback), is what happens if the server-side Bolt admin configuration has disabled pre-auth.  

In this instance, it is not just the two lines that should be removed, but actually the entire `case` for `Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_PRODUCT_PAGE`.

Instead, it should rely on the `default` `case`.

**Pull request merged**